src/pages/contacts/index.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import { Container, Title } from "@/styles/components";
+import { useAppSelector } from "@/store";
+import {
+    ContactCard,
+    ContactCardText,
+    ContactLink,
+    ContactsCards,
+    ContactsPageWrapper,
+    ContactsSocials,
+} from "./style";
+import PhoneIcon from "@/components/global/icons/contacts/PhoneIcon/PhoneIcon";
+import MailIcon from "@/components/global/icons/contacts/MailIcon/MailIcon";
+import AddressIcon from "@/components/global/icons/contacts/AddressIcon/AddressIcon";
+import { Socials } from "@/components/menus/Socials";
+import theme from "@/styles/theme";
+import { useRouter } from "next/router";
+import Breadcrumbs from "@/components/global/Breadcrumbs/Breadcrumbs";
+import ContactsForm from "@/components/pages/contacts/ContactsForm/ContactsForm";
+import { useTranslations } from "next-intl";
+import { useResponsive } from "@/hooks/useResponsive";
+import { Skeleton } from "@mui/material";
+
+const ContactsPage = () => {
+    const t = useTranslations("Contacts");
+    const tBreadcrumbs = useTranslations("Breadcrumbs");
+    const { isTablet } = useResponsive();
+    const themeOptions = useAppSelector((state) => state.themeOptions);
+    const ContactItems = themeOptions.data.item.contacts;
+
+    //which  way for  breadcrumbs is right?
+    //1
+    const router = useRouter();
+    const currentPath = router.asPath;
+    const pathParts = currentPath.split("/").filter((part) => part);
+    const breadcrumbsLinks = pathParts.map((part, index) => {
+        const url = `/${pathParts.slice(0, index + 1).join("/")}`;
+        return { name: part.charAt(0).toUpperCase() + part.slice(1), url };
+    });
+    //2
+    const breadcrumbsLinksMock = [
+        { name: tBreadcrumbs("homePage"), url: "/" },
```

Comment:
**Approach 1** is suitable if you have many pages and want to automatically generate breadcrumbs based on the current path.
**Approach 2** is suitable if you have fixed pages and want full control over display names and URLs, as well as use localization.

For us, the **second** option is preferable.